HISTORY.md

```diff
@@ -8,6 +8,7 @@
 * Fix a bug when format_version=3, partitioned fitlers, and prefix search are used in conjunction. The bug could result into Seek::(prefix) returning NotFound for an existing prefix.
 * Revert the feature "Merging iterator to avoid child iterator reseek for some cases (#5286)" since it might cause strong results when reseek happens with a different iterator upper bound.
 * Fix a bug causing a crash during ingest external file when background compaction cause severe error (file not found).
+* Fix a bug when partitioned fitlers and prefix search are used in conjunction, ::SeekForPrev could return invalid for an existing prefix. ::SeekForPrev might be called by the user, or internally on ::Seek followed by ::Prev, or within Seek if the return value involves Delete or a Merge operand.
```

Comment:
I don't know whether users got the feeling that Seek + Next + Prev won't have the bug. It will be good to clarify.

Reply:
I extended it to all ::Prev calls since Seek does not have be called to be a forward direction iterator.